sopel/modules/version.py

```diff
@@ -11,6 +11,7 @@
 
 from datetime import datetime
 import sopel
```

Comment:
In the spirit of the following import, maybe make this `from sopel import __version__ as release` or similar? Just to avoid importing the entire `sopel` namespace for access to one property.

Reply:
Certainly. It was present initially, and I didn't want to touch too much more than was necessary. I'll push an additional commit within an hour.